### DISCUSSION TITLE
Validate against input type for filters

### DISCUSSION THREAD

**Discussion by oreilles** [directus/directus#12691] [Feature Requests]
When setting a `eq`/`neq` filter on a `uuid` field and comparing it to something that cannot be parsed as a `uuid`, the database returns some error like `invalid input syntax for type uuid: "foo"`.
This is not handled appropriately, and therefore the app shows the "Unexpected error" info box.

### To Reproduce

Set an `eq` filter on a `uuid` field, comparing it to a non uuid-like string.

**Comment by oreilles** [comment]
Question: is this still needed ? The application won't send invalid uuid anymore (https://github.com/directus/directus/pull/8696/).

**Comment by rijkvanzanten** [comment]
Yeah, we should validate this API side 👍🏻

**Comment by rijkvanzanten** [comment]
This is being tracked and implemented under https://github.com/directus/directus/issues/3612

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
